### ISSUE TITLE
start_time is required in Azure Subscription Utilization Records (BUG)

### ISSUE THREAD

**Issue by vidarw** [PartnerCenterSamples/Partner-Center-SDK-Samples#11]
Using the unaltered example from 16 - 1 _Retrieve Azure Subscription Utilization Records_.

The error message is attached below:

```
Partner Exception:
Error Category: BadInput
Service Error Payload:
Error code: 2001
Error message: start_time is required
Error data:
start_time


Context: Request Id: 3f7c36f4-b16f-428c-9bbe-9c70d3d3338f, Correlation Id: d3967dd7-e79f-4626-a566-ccb81f0f1156, Locale: en-US
Base Description: Microsoft.Store.PartnerCenter.Exceptions.PartnerException: start_time is required
   at Microsoft.Store.PartnerCenter.PartnerService.SynchronousExecute[T](Func`1 operation)
   at Microsoft.Store.PartnerCenter.Utilization.AzureUtilizationCollectionOperations.Query(DateTimeOffset startTime, DateTimeOffset endTime, AzureUtilizationGranularity granularity, Boolean showDetails, Nullable`1 size)
   at Microsoft.Store.PartnerCenter.Samples.Utilization.GetAzureSubscriptionUtilization.RunScenario() in C:\Projects\Partner-Center-SDK-Samples\Source\Partner Center SDK Samples\Utilization\GetAzureSubscriptionUtilization.cs:line 37
   at Microsoft.Store.PartnerCenter.Samples.BasePartnerScenario.Run() in C:\Projects\Partner-Center-SDK-Samples\Source\Partner Center SDK Samples\BasePartnerScenario.cs:line 78
```

Request extracted from fiddler:
`https://api.partnercenter.microsoft.com/v1/customers/91FAFCD9-XXXX-XXXX-XXXX-A8F2C544F835/subscriptions/C7F6819F-XXXX-XXXX-XXXX-8F2DFB6BF65D/utilizations/azure?start_time=2016-03-01T16.26.07Z&end_time=2017-03-01T16.26.07Z&granularity=Daily&show_details=True&size=100`

Changing the dots (.) from 2016-03-01T16.26.07Z into colon (:) 2016-03-01T16:26:07Z and then url encode to 2016-03-01T16%3A26%3A07Z solves the problem. Computer locale/region is Norwegian/Norway.

**Comment by alidhia** [comment]
Hi Vidar,
do you mind providing some repro steps, I tried to replicate your issue with no luck.
was the customer's region you are trying to fetch utilization records for in Norway as well?

**Comment by vidarw** [comment]
Checked out f355b71
Open project in VS2015 Update 3 addded details to App.config and built source (with nuget restore)
Run/Debug application and choose step 16-1
Enter tenant and subcription ID.
Descirbed error occurs.

Yes. Machine is located in Norway.
Changing from "_Norwegian_" to "_English (United States)_" or "_English (Europe)_" in "_Change date, time or number formats_" resolves the issue.

**Comment by alidhia** [comment]
Thanks Vidar!
I was able to repro and a bug has been created for this issue. Thank you very much for bringing this issue to our attention!

**Comment by alidhia** [comment]
This should be fixed in v 1.4.0.
Thanks
